experimental/auth-logic/test_data/oak_ec_private.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN EC PARAMETERS-----
```

Comment:
Well this key is not actually secret, so it is fine (in fact expected) to be checked in. Otherwise how would we use it to sign additional things for testing?

Reply:
If we want to try something different in the json files, we would have to sign them again according to the instructions in experimental/auth-logic/test_data/rekor_entry_instructions.md , and we wouldn't want to regenerate the keys just for that

Reply:
Sounds good.